src/lib/utilities/decode-payload.test.ts

```diff
@@ -142,3 +156,37 @@ describe(convertPayloadToJson, () => {
     expect(dataConverterStatus).toEqual('notRequested');
   });
 });
+
+describe(convertPayloadToJsonWithCodec, () => {
```

Comment:
Should this be commented out?

Reply:
Nope, need to mock out fetch within that function.